animepahe-dl.sh

```diff
@@ -262,12 +262,13 @@ main() {
     set_args "$@"
     set_var
 
-    if [[ -z "${_ANIME_SLUG:-}" ]]; then
+    if [[ ! -s ./anime.list ]]; then
         download_anime_list
-        [[ ! -s "$_ANIME_LIST_FILE" ]] && print_error "$_ANIME_LIST_FILE not found!"
-        _ANIME_SLUG=$($_FZF < "$_ANIME_LIST_FILE" | awk -F']' '{print $1}' | sed -E 's/^\[//')
     fi
```

Comment:
1) Please use variable `$_ANIME_LIST_FILE` instead of `./anime.list`
2) This `if` will cause the issue when new amines added on the server, if someone used script for a while and had `anime.list` downloaded, then new amines won't be downloaded.

Reply:
1. Ok.
2. Of course, that's why we need an update (`-u`) flag that cleans (removes) `anime.list` as well as `.source.json`s. (Also, it would be nice if directories are `mkdir`ed inside a separate/dedicated directory.)

Question: do slugs expire or only URLs obtained by `-l` flag?